wordpress-mobile/AztecEditor-iOS
# Issue/refactor demo app

Addresses #1224 

This is a first step on simplifying the EditorDemoController, by breaking it in smaller classes.
On this PR we mainly focus around the attachment handling by creating two classes:
 - MediaInserter: to do the actions to insert media object in the TextView
 - TextViewAttachmentDelegateProvider: to provide all the delegate methods and actions around the attachment handling on TextView

To test:
 - Check the demo app and see that all media bevahiour is correct:
  - insert Image
  - insert Video
  - Tap on Image/Video and see if the actions show correctly.
  - Tap on unknown HTML and see if the actions show correctly.
